common/vts.proto

```diff
@@ -11,11 +11,9 @@ import "attestation.proto";
 import "token.proto";
 
 
-enum TAType {
```

Comment:
As you are tidying up vts.proto, I realized the following can also be tidied up quite easily.

Line: 21: There is no EndorsementID

Line 30: GetEndorsementsRequest() is NOT used any more [ I confirmed this with @setrofim : Thanks]

Line 40: GetEndorsementsResponse() is NOT used either (on the interface)

Reply:
I think we should keep this PR scoped to what it says.

Reply:
Now that you are using the term Endorsement in your changes, this becomes extremely cluttered and confusing!

Reply:
Anyway, I'll do this :)

Reply:
Anyway, I will do this post the merge of this one: See Issue #150 to handle this!

Reply:
No need, it's already done in [31e8daa](https://github.com/veraison/veraison/pull/148/commits/31e8daae294514b9cea532bb14f0786e50f0e9d8)